homeassistant/components/ffmpeg.py

```diff
@@ -96,12 +97,38 @@ def __init__(self, hass, ffmpeg_bin):
         self.hass = hass
         self._cache = {}
         self._bin = ffmpeg_bin
+        self._version = None
 
     @property
     def binary(self):
         """Return ffmpeg binary from config."""
         return self._bin
 
+    async def async_get_version(self):
+        """Return ffmpeg version."""
+        from haffmpeg.tools import FFVersion
+
+        if self._version is None:
+            ffversion = FFVersion(self._bin, self.hass.loop)
+            self._version = await ffversion.get_version()
+
+        return self._version
```

Comment:
Call that function on setup. After that, you don't need a coro later and it will be a lot faster